environment-dev.yml

```diff
@@ -28,6 +28,7 @@ dependencies:
   - python=3.10
   - pyupgrade=3.1.0
   - terraform=1.3.4
+  - openjdk=11
```

Comment:
Out of curiosity, why 11? This is neither our outdated Java 8 nor the latest.

Reply:
It was what was automatically installed when I added "openjdk" with conda. 

I believe openjdk 8 doesn't work with our engine anyways, so that's why  it's not at java 8.

Reply:
Ah actually it looks like openapigenerator has a minimum of java 11 and may require java 11 for mac users

https://github.com/OpenAPITools/openapi-generator#13---download-jar